pkg/install/deployment.go

```diff
@@ -152,6 +152,11 @@ func Deployment(namespace string, opts ...podTemplateOption) *appsv1.Deployment
 				Key:      "node-role.kubernetes.io/master",
 				Operator: "Exists",
 			},
+			corev1.Toleration{
+				Effect:   "NoSchedule",
+				Key:      "node-role.kubernetes.io/control-plane",
+				Operator: "Exists",
+			},
```

Comment:
Do we still need to keep lines 150-153?

Reply:
We need to keep both to support backward compatibility. You can see https://github.com/kubernetes/enhancements/tree/master/keps/sig-cluster-lifecycle/kubeadm/2067-rename-master-label-taint for more information.